ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/manager/EthServerTest.java

```diff
@@ -43,13 +42,16 @@
   private final EthPeer ethPeer = mock(EthPeer.class);
   private final EthMessages ethMessages = new EthMessages();
 
-  @Before
-  public void setUp() {
-    new EthServer(blockchain, worldStateArchive, ethMessages, 2);
-  }
-
   @Test
   public void shouldRespondToNodeDataRequests() throws Exception {
+    new EthServer(
```

Comment:
Why not just set up a single instance in the `setUp`?

Reply:
Can't test differents limits with single instance. Some tests would fail